mautic/mautic
# [UX] Better description for existing dashboard templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR ensures that all default dashboard templates come with a description following this structure:
- How many widgets we have in it
- What is this template about

Example:
"Start with a dashboard of 6 reports that show the kind of requests your team receives over email."




| Before                                 | After
| -------------------------------------- | ---
|  ![image](https://github.com/user-attachments/assets/b4b00f3e-c7a2-4c0b-b589-5a88f850e5bd)                                      | ![image](https://github.com/user-attachments/assets/05f908e2-3e2b-4ee3-8fc5-49eeb5baf1c3)




---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the Dashboard, click on the page actions and enter to see the pre-defined dahboards.
3. Read each description.
